homeassistant/scripts/check_config.py

```diff
@@ -325,7 +325,7 @@ def _comp_error(ex, domain, config):
     # Merge packages
     merge_packages_config(
         hass, config, core_config.get(CONF_PACKAGES, {}), _pack_error)
-    del core_config[CONF_PACKAGES]
+    core_config.pop(CONF_PACKAGES, None)
```

Comment:
Can you make the previous line `.pop` iso `.get`?

Reply:
I tried popping the packages from core config, but then we get a key error on line 296 in `_pack_error` in the tests. A refactor could probably clean this up, but I don't think that belongs in this PR.